log-manager/src/main/java/io/airlift/log/LoggingConfiguration.java

```diff
@@ -155,4 +156,16 @@ public LoggingConfiguration setLegacyLoggerImplementationEnabled(boolean enabled
         this.legacyLoggerImplementationEnabled = enabled;
         return this;
     }
+
+    public String getSystemContextFile()
```

Comment:
What's a "system context"?

Reply:
It's a file that we can pass values in from kubernetes, either through the helm chart or environment variables in the container, that we add onto our logs so we can eliminate the need for fluentbit or fluentd to instrument the logs with data we know when we start the application. In particular, things like service name, pod name, ip address, etc. are valuable for debugging.